app/src/main/java/com/imashnake/animite/features/media/MediaUiState.kt

```diff
@@ -6,9 +6,30 @@ data class MediaUiState(
     val color: String? = "",
     val title: String? = "",
     val description: String? = "",
-    val averageScore: Int? = null,
-    val ranks: MutableList<Pair<String, Int>> = mutableListOf(),
-    val genres: List<String?>? = listOf(),
-    val characters: List<Pair<String?, String?>>? = listOf(),
-    val trailer: Pair<String?, String?> = Pair("", "")
+    val stats: MutableList<Stat> = mutableListOf(),
```

Comment:
Mutability might violate the single source of truth principle (see the [Android docs on app architecture](https://developer.android.com/topic/architecture#single-source-of-truth)) as you won't be able to know what changed it, and you might not be notified of it if it does